core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -145,6 +145,17 @@ private[spark] object UIUtils extends Logging {
     }
   }
 
+  // Format raw bytes to binary prefix byte units
+  def formatBytesBinary(bytes: Long): String = {
```

Comment:
Doesn't `Utils.bytesToString` do this?

Reply:
@abellina and @srowen Yes you both are right, we do not need a new method. Have updated the PR. Thank you.